.lintstagedrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "*.ts": ["prettier --write", "lint --fix", "git add"],
```

Comment:
Why was the `lint --fix` removed?

Reply:
Has 0 impact and fails in 90% of cases when running via `lint-staged`. `yarn lint --fix` returns 0 code and `lint --fix` (via lint-staged) throws with unknown reasons.